### ISSUE TITLE
Parser: omit occurrence zero

### ISSUE THREAD

**Issue by nichtich** [deutsche-nationalbibliothek/pica-rs#298]
Occurrence zero is equivalent to no occurrence, so don't include occurrence value zero. For instance:

    $ echo -e "001A/00 \x1fab\x1e" | pica print
    001A $ab
    $ echo -e "001A \x1fab\x1e" | pica print
    001A $ab

**Comment by niko2342** [comment]
I don't like to enable this by default, because we lose the view to underlying data and it's not possible to distinguish between zero occurrence and no occurrence.

What do you think about implementing a CLI and config flag (`--omit-zero-occurrence`)?

**Usage:**

```bash
$ # zero occurrence are printed by default
$ echo -e "001A/00 \x1fab\x1e" | pica print
001A/00 $ab

$ # zero occurrences can be omitted by cli flag
$ echo -e "001A/00 \x1fab\x1e" | pica print --omit-zero-occurrence
001A $ab

$ # zero occurrences can be omitted by config
$ cat Pica.toml
[print]
omit-zero-occurrence = true

$ echo -e "001A/00 \x1fab\x1e" | pica --config Pica.toml print
001A $ab
```

**Comment by nichtich** [comment]
> and it's not possible to distinguish between zero occurrence and no occurrence

That's the purpose :-) Internally *every* field has an occurrence but zero-occurrence are not shown. Semantically both are the same and they won't be distinguishable once read into CBS database.

> because we lose the view to underlying data

Then add a flag to explicitly retain zero occurrences in the parser, not in the printer.

**Comment by niko2342** [comment]
Thanks for the clarification. I'll change the parser logic to ignore zero-occurrences.

**Comment by niko2342** [comment]
This feature will be included in the next release (v0.7.0).
